src/common/remote/github/extractPullRequestIdentifier.ts

```diff
@@ -1,6 +1,7 @@
 import { PullRequestIdentifier } from './types';
 
 export const extractPullRequestIdentifier = (identifier: string): PullRequestIdentifier => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const [owner, _, repo, __, prId] = identifier.split(/(\/|#)/);
```

Comment:
[Note] _ and __ were unused. Am I good to investigate removing them?

Reply:
I think we can remove them. This was a community contribution and some languages like Go enforce you restructure the whole object as once. I see no need for it here.

Reply:
Have removed and it seems to make no difference